incubator/vamp_install/script.md

```diff
@@ -75,6 +75,8 @@ the CLI:
 dcos marathon app add elasticsearch.json
 ```
 
+*At the time of writing, Bash on Windows was unable to connect to forwarded localhost port. In this case, the DC/OS web interface can be used to start Elastic Search. Navigate to Services -> Run a Service -> JSON Configuration and paste the contents of `elasticsearch.json`*
```

Comment:
This breaks the demo flow for automated demos and thus is not an acceptable workaround. It's OK to leave this in here but we need something that will work in an automated flow. This should therefore add something like "In order to run in a fully automated fashion be sure to use Cygwin or Mingw if using windows, which do not have this limitation".